.github/workflows/ci-android-jni.yml

```
@@ -51,7 +51,7 @@ jobs:
           distribution: "zulu"
           java-version: 11
```

If you install Java 17 instead of 11 this PR most likely works:
```suggestion
          java-version: 17
```